tests/x11regressions/firefox/firefox_rss.pm

```diff
@@ -38,6 +38,7 @@ sub run {
     send_key "alt-d";
     type_string "https://linux.slashdot.org/\n";
     $self->firefox_check_popups;
+    assert_and_click("slashdot-cookies-agree") if check_screen("slashdot-cookies", 0);
```

Comment:
In the verification runs I cannot see this code working because cookies agreement isn't shown. But I also don't think we need it, as subscription will still work even if agreement is not accepted, so I suggest just to create a needle for clean firefox, so we proceed without code changes.